docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md

```diff
@@ -0,0 +1,362 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: Using the Linode Longview App for Apache
+keywords: ["Longview", " Apache", " statistics", " mod\\_status"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+aliases: ['longview/longview-for-apache/', 'platform/longview/longview-app-for-apache/']
+modified: 2018-12-26
+modified_by:
+  name: Linode
+published: 2013-11-04
+title: What is the Linode Longview App for Apache
+h1_title: Using the Linode Longview App for Apache
+external_resources:
+ - '[cPanel Products News](https://news.cpanel.com/category/products/)'
+ - '[cPanel Security News](https://news.cpanel.com/category/security/)'
+ - '[Unix Toolbox](http://cb.vu/unixtoolbox.xhtml)'
+---
+
+Longview for Apache is a Longview App. The Longview Apache tab appears in the Linode Cloud Manager when Longview detects that you have Apache installed on your Linode. With the Longview Apache App, you'll be able to view statistics for Apache on your Linode. It can help you keep track of Apache's settings, workers and requests, system resource consumption, and other information.
+
+## In this Guide:
+
+This guide discusses Linode Longview for Apache. The guide covers the following topics:
+
+- [Configuring Linode Longview for Apache](#debian-and-ubuntu-automatic-configuration).
+- [Interacting with the Apache data provided by Longview in the Linode Cloud Manager](#viewing-statistics).
+- [Troubleshooting Linode Longview for Apache](#troubleshooting).
+
+## Before you Begin
+
+In order to use the Longview App for Apache, ensure you have completed the following things:
+
+-  A Linode with [Apache installed and running]((/docs/web-servers/apache/)).
```

Comment:
Remove extra set of parenthesis in the link

Reply:
done